packages/online-editor/src/editor/DmnRunner/DmnRunnerDrawerPanelContent.tsx

```diff
@@ -322,16 +318,15 @@ export function DmnRunnerDrawerPanelContent(props: Props) {
   const onAddNewRow = useCallback(() => {
     dmnRunnerDispatch.setInputRows((previousData: Array<InputRow>) => {
       const newData = [...previousData, {}];
-      dmnRunnerDispatch.setCurrentInputRowIndex(newData.length - 1);
+      dmnRunnerDispatch.setParams(DmnRunnerMode.FORM, true, newData.length - 1);
```

Comment:
Floating boolean argument does not make code very readable. Please consider giving it an object parameter. Like this:

```suggestion
      dmnRunnerDispatch.setParams({ mode: DmnRunnerMode.FORM, expand: true, row: newData.length - 1 });
```